frog-utils/src/types.js

```diff
@@ -275,6 +275,7 @@ type LIRenderPropsT = {|
 |};
 
 export type LIRenderT = React.ComponentType<LIRenderPropsT>;
+type ImmutableLIT = { id: string, payload: Object };
```

Comment:
Should be named something else that payload as it is not the payload of the LI. This will be confusing.
```
{
  "id": "cjhlndsqo00063i6jw0h924z2",
  "payload": { 
    "liType": "li-image", 
    "payload": { 
      "url": "/file?name=cjhlndsb100053i6ji1pzpdj1", 
      "thumburl": "/file?name=cjhlndsb100053i6ji1pzpdj1thumb" 
    },
    "createdAt": "2018-05-25T07:31:22.176Z" 
  } 
}
```